Tests/ComparisonTest.cs

```diff
@@ -160,30 +160,49 @@ public static string EvaluatorPrepareEnvironmentCode(Dictionary<string, string>
 
             JSFilenames = null;
 
+            AssemblyAppDomain = AppDomain.CreateDomain("TestAssemblyDomain", null, new AppDomainSetup
```

Comment:
If we pass here AppDomain.CurrentDomain, it will work in same domain. We could introduce some options later to make it run faster (but with more memory usage/state effects).